### ISSUE TITLE
Error when Installing from Source - javadoc: error - invalid  flag: -XDoclint:none 

### ISSUE THREAD

**Issue by mbp-informatics** [openmicroscopy/openmicroscopy#5001]
I have been trying to install OMERO from source on an Ubuntu 16.04 VM with Ice 3.6.3 following the walk throughs at:

http://www.openmicroscopy.org/site/support/omero5.2/sysadmins/unix/server-ubuntu1404-ice36.html
http://www.openmicroscopy.org/site/support/omero5.2/developers/installation.html

I ran into an error executing the build.py script to build the OMERO server after installing all of the dependencies following the walk throughs. The error I received is as followed:

```
Generating Javadoc
Javadoc execution
javadoc: error - invalid flag: -Xdoclint:none

BUILD FAILED
/home/mbpadmin/openmicroscopy/build.xml:178: The following error occurred while executing this line:
/home/mbpadmin/openmicroscopy/components/antlib/resources/global.xml:430: The following error occurred while executing this line:
/home/mbpadmin/openmicroscopy/components/antlib/resources/lifecycle.xml:400: Javadoc returned 1
```

I did some research and I could only find information that the error could occur if I was running Java 1.7, as the -Xdoclint:none flag  exists for Java 1.8 only. But I downloaded Java 1.8 as the dependency, so I believe that should not be the problem. I found a post on the OMERO boards, that mentioned that OMERO could not be built from source with ice 3.6.3 and thought it might be related. The post is found at:

https://www.openmicroscopy.org/community/viewtopic.php?f=5&t=8143&p=17555&hilit=source#p17555

**Comment by jburel** [comment]
We released version 5.2.7 yesterday, version adding support for ice 3.6.3
Could you try to run ``ant build-default -Dpackage-extra=false``? ``package-extra=false`` means that the javadoc won't be built

**Comment by jburel** [comment]
Alternatively, you could install the released version http://downloads.openmicroscopy.org/omero/5.2.7/
I have just installed OMERO v5.2.7
on Ubuntu 16.04 with java version  "1.8.0_111" (openjdk), ice 3.6.3 and posgresql 9.4
